Gemfile.lock

```diff
@@ -42,60 +42,63 @@ GEM
       rspec-expectations (>= 2.99)
       thor (>= 0.19, < 2.0)
     ast (2.4.0)
-    backports (3.15.0)
-    bcrypt_pbkdf (1.0.1)
+    backports (3.18.1)
+    bcrypt_pbkdf (1.1.0.rc1)
     builder (3.2.4)
-    chef (15.7.30)
+    chef (16.2.44)
```

Comment:
We re-resolve the dependencies for Chef Workstation via its master Gemfile.lock. I see no issue with updating this dependency here, because Chef Workstation uses Berkshelf as a library, and my understanding is that Chef Workstation is how most users consume Berkshelf.